cypress/e2e/tests/migration/migration-waves/sort.test.ts

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    sortAsc,
+    sortDesc,
+    verifySortAsc,
+    verifySortDesc,
+    getTableColumnData,
+    deleteByList,
+    createMultipleMigrationWaves,
+    verifyDateSortAsc,
+    verifyDateSortDesc,
+} from "../../../../utils/utils";
+import { startDate, endDate } from "../../../types/constants";
+
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+import { name } from "../../../types/constants";
+import { fieldHeader } from "../../../views/stakeholders.view";
+import { MigrationWaveView } from "../../../views/migration-wave.view";
+
+let migrationWavesList: Array<MigrationWave> = [];
+
+//Automates Polarion TC 341
+describe(["@tier2"], "Migration Waves sort validations", function () {
+    before("Login and Create Test Data", function () {
+        // Perform login
+        login();
+
+        // Create multiple Migration Waves
+        migrationWavesList = createMultipleMigrationWaves(2);
+    });
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+
+        // Interceptors
+        cy.intercept("GET", "/hub/migration-waves*").as("getMigrationWaves");
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the Migration Waves created before the tests
+        deleteByList(migrationWavesList);
+    });
+
+    it("Name sort validations", function () {
+        // Navigate to Migration Waves tab
+        MigrationWave.open();
+        cy.get("@getMigrationWaves");
+        // get unsorted list when page loads
+        const unsortedList = getTableColumnData(name);
+
+        // Sort the Migration Waves by name in ascending order
+        sortAsc(name, MigrationWaveView.fieldHeader);
+        cy.wait(2000);
```

Comment:
Can you try to remove all `cy.waits` from this test suite and check if the result is the same?

If the test doesn't work with the `cy.wait` try to reduce the waiting time as much as you can, thanks